scripts/validations.py

```diff
@@ -0,0 +1,125 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
```

Comment:
It doesn't seem like checking for `None` is any safer, since we're using the same value regardless (and I don't see this being mutated). Maybe I'm missing something...